dpgen/data/surf.py

```diff
@@ -440,15 +441,12 @@ def make_scale(jdata):
         for jj in scale:
             if skip_relax:
                 pos_src = os.path.join(os.path.join(init_path, ii), "POSCAR")
-                assert os.path.isfile(pos_src)
```

Comment:
Why remove this assert? I don't understand.

Reply:
If you read some more lines, you will is that assert is redundant